UnicodeJsps/src/main/java/org/unicode/jsp/CachedProps.java

```diff
@@ -44,7 +44,7 @@
     final BiMultimap<String,String> nameToAliases = new BiMultimap<String,String>(null,null);
     final Map<String,BiMultimap<String,String>> nameToValueToAliases = new LinkedHashMap();
 
-    static CachedProps CACHED_PROPS = getInstance(VersionInfo.getInstance(12));
+    static CachedProps CACHED_PROPS = getInstance(VersionInfo.getInstance(14));
```

Comment:
It's a bit more complicated than that because of the interplay with the beta flag. I suggested that we go with 14 for now, and file an issue. We don't want to wait on the fuller solution.